docs/concepts.md

```diff
@@ -23,10 +23,18 @@ The high level architecture of the operator is as follows:
 
 Each approach have their advantages and disadvantages. 
 
-The kubernetes, grpc and flagd-proxy sync configuration has the advantage of providing near real-time flag updates(on the order of seconds) to the flagd sidecar. 
-
-For example, Kubernetes syncs require the flagd sidecar(and consequently the workload pod) to communicate with the 
-Kubernetes API. This may violate the security or network policies of some organizations.
+The `kubernetes`, `grpc` and `flagd-proxy` sync configuration has the advantage of providing near real-time flag updates(on the order of seconds) to the flagd sidecar. 
+
+The `kubernetes` syncs require the flagd sidecar(and consequently the workload pod) to communicate with the 
+Kubernetes API. The OpenFeature Operator registers a [mutating admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
+that changes the manifest of Pods annotated with [OpenFeature annotations](./annotations.md) and adds a `ClusterRoleBinding`
+to the `ServiceAccount` of the Pod. This may violate the security or network policies of some organizations.
+Furthermore, if you're using GitOps to deliver your application, please ensure that the `ClusterRoleBinding` deployed with the application
```

Comment:
Prefer this in a new paragraph to give more prominence

Reply:
```suggestion

Furthermore, if you're using GitOps to deliver your application, please ensure that the `ClusterRoleBinding` deployed with the application
```